packages/external-db-config/lib/readers/authorization_config_reader.js

```diff
@@ -0,0 +1,20 @@
+const { checkRequiredKeys, isJson, EMPTY_ROLE_CONFIG } = require('../utils/config_utils')
+
+class AuthorizationConfigReader {
+  constructor() {
+  }
+
+  async readConfig() {
+    const { ROLE_CONFIG } = process.env
+    const { collectionLevelConfig } = isJson(ROLE_CONFIG) ? JSON.parse(ROLE_CONFIG) : EMPTY_ROLE_CONFIG
```

Comment:
what happens in case the json is not parsing or contain bad data ?

Reply:
Empty role config.
Default permissions for all the collections

Reply:
user can provide invalid json and not be aware of that.
what i mean is a valid json, but an invalid data.
we need to validate it as well and inform the user of that.

Reply:
WDYT that we need to validate for example? 
that the JSON is in the right format (`id`,` readPolicies`,` writePolicies`) ?